DeepMRSeg/deepmrseg_test.py

```diff
@@ -506,6 +508,8 @@ def _main( argv ):
 	_sys.stdout.flush()
 # ENDDEF MAIN
 
+def runFromCMD():
```

Comment:
This function is not required. Call _main directly from setup.py

Reply:
done. Need to test with data.